.github/workflows/ci.yaml

```diff
@@ -24,6 +24,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       CC: deny_c
+      # we want to build r-a on stable to check that it keeps building on stable,
+      # but we also want to test our proc-macro-srv which depends on nightly features
+      RUSTC_BOOTSTRAP: 1
```

Comment:
I think you can set `RUSTC_BOOTSTRAP` to a set of crates like `RUSTC_BOOTSTRAP=proc-macro-srv`?